brave/browser-laptop
# show message when no payments have been made

Fix #6555

Auditors @bsclifton @mrose17 @bradleyrichter 

## Test Plan

1. Go to Preferences#payments
2. Make sure it now reads "Your first payment will be in".. some amount of time
3. Simulate a payment
4. Make sure it now reads "Your last payment was"... some amount of time

<img width="935" alt="screen shot 2017-01-11 at 9 32 17 pm" src="https://cloud.githubusercontent.com/assets/490294/21878240/ca1ac7ac-d845-11e6-92bc-e857e2a723da.png">
